acc_test/resource_kafka_test.go

```diff
@@ -41,16 +44,26 @@ func TestKafkaResources(t *testing.T) {
 	configBytesAcl2, _ := ioutil.ReadFile("data/kafka_acl_create_acl_duplicate.tf")
 	configBytesAcl3, _ := ioutil.ReadFile("data/kafka_acl_list.tf")
 
+	// config bytes for Kafka Connect
+	kcConfigBytes, _ := ioutil.ReadFile("data/kafka_connect_create_cluster.tf")
+
 	kafkaUsername1 := "charlie1"
 	kafkaUsername2 := "charlie2"
 	kafkaUsername3 := "charlie3"
 	oldPassword := "charlie123!"
 	newPassword := "charlie123standard!"
 	kafkaNodeSize := "KFK-DEV-t4g.medium-80"
 	zookeeperNodeSize := "KDZ-DEV-t4g.small-30"
-	kafkaVersion := "2.7.1"
+	kafkaVersion := "apache-kafka:3.0.0.ic1"
+	kcVersion := "kafka-connect:3.0.0.ic1"
 	topic1 := "test1"
 	topic2 := "test2"
+	azureStorageAccountName := os.Getenv("IC_AZURE_STORAGE_ACCOUNT_NAME")
```

Comment:
Are these all already added to CircleCI

Reply:
not yet, hence all the discussions before about where this is gonna be put. I think along with this deployment, I will be adding the values to the CircleCI env

Reply:
ok, don't forget to record themeless, because once they're in circleCI you won't be able to see them again